docs/core/compatibility/6.0.md

```diff
@@ -13,6 +13,7 @@ If you're migrating an app to .NET 6.0, the breaking changes listed here might a
 ## ASP.NET Core
 
 - [Nullable reference type annotations changed](aspnet-core/6.0/nullable-reference-type-annotations-changed.md)
+- [Obsoleted and removed APIs](aspnet-core/6.0/obsolete-removed-apis.md)
```

Comment:
There's an additional set of modified APIs at https://github.com/aspnet/Announcements/issues/446.

Reply:
Those APIs were already documented in a separate doc: https://docs.microsoft.com/dotnet/core/compatibility/aspnet-core/6.0/razor-engine-apis-obsolete?branch=pr-en-us-22862